Remotion/Web/Core/Res/Themes/NovaViso/HTML/SingleView.css

```diff
@@ -86,4 +86,5 @@ div.singleViewBottomControls
 div.singleViewBottomControls.empty
 {
   box-shadow: none;
+  display: none;
```

Comment:
> [....] Otherwise, there would be no border and just a 1em padding. [...]
>
> Please update the ticket with info that the border-display isn't fixed by this bugreport.

What border-display? I cannot find anything related to a border in the issue description or attached screenshot. 

> Also: the actuall issue of the reporter isn't solved with this fix since the area isn't identified as 'empty'.

I'll reject the issue in the current state and ask for more information about the page state and current dom structure when the issue appears.

Reply:
Yes, I was blind. The re-motion ticket requests that the bottom controls should not take any space and do not have a border, if the `empty` css class is applied. Contrary to the ticket text, this also affects TabbedMultiView. Any other effects in downstream projects are not relevant for this bug report but separate issues.